airbyte-integrations/connectors/source-partnerstack/source_partnerstack/spec.yaml

```diff
@@ -0,0 +1,27 @@
+documentationUrl: https://docs.partnerstack.com/reference/auth
```

Comment:
Change to: `https://docs.airbyte.com/integrations/sources/partnerstack`

Reply:
✅